lib/screen/group_and_profile_screen.dart

```diff
@@ -0,0 +1,217 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+import '../widgets/my_navigation_bar.dart';
+import '../widgets/outlined_button.dart';
+
+class ProfileScreen extends StatelessWidget {
+  const ProfileScreen(this.isGroup, {super.key});
```

Comment:
`isGroup` should be a named parameter instead of positional by convention (and better for self-documenting):
```suggestion
  const ProfileScreen({super.key, this.isGroup});
```